app/public/js/localSprites.js

```diff
@@ -134,7 +162,7 @@ function updateLocalSprite(localSprite) {
     //WRONG: TOO SPECIFIC TO FIREBALL RIGHT NOW. It seems like you could give addSprite functions a bunch of new parameters to fix this, but that seems messy, as a lot of sprites won't have contrails.
     if (localSprite.hasContrail === true) {
         if (localSprite.contrailTimer >= localSprite.framesBetweenContrailParticles) {
-            addFireballParticle(localSprite, 30, 32, 32, 0); //for last argument, 0 = contrail, 1 = detonation
+            addFireballParticle(localSprite, 30, 32, 32, SPRITE_TYPE_FIREBALL_PARTICLE_CONTRAIL);
```

Comment:
I think this diff perfectly illustrates the value of using these constants.

Reply:
Also, we can probably rename this to something like `addChildSprite` now, since it should work fine for non fireball things too right?